web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,55 @@
+# Generating Diagnostic Report
+
+Users can download a cluster diagnostic report based on must-gather tool for upload to Red Hat Support or their own diagnostics.
+
+## Accessing 'Generate Report' actions
+
+There are several ways the user can access the `Generate (diagnostics) report` action.
+
+### Via the masthead help menu
+![Masthead help click](img/1-1a-clusterdash-startdownload.png)
+![Masthead help menu](img/1-1b-clusterdash-startdownload.png)
+
+### Via the cluster dashboard status card kebab menu
+![Cluster dashboard status card kebab click](img/1-2a-clusterdash-startdownload.png)
+![Cluster dashboard status card kebab menu](img/1-2b-clusterdash-startdownload.png)
+
+### Via a cluster dashboard status card message action
+![Cluster dashboard status card message click](img/5-1-clusterdash-CNVerror.png)
+- The cluster dashboard will soon support warning messages to notify the user that CNV or OCS are degraded. From these messages the user will be able to immediately `Generate report` using the action link within the message.
```

Comment:
```suggestion
- In the future, the cluster dashboard will support warning messages to notify the user that CNV or OCS are degraded. From these messages the user will be able to immediately `Generate report` using the action link within the message.
```

Reply:
We might want to remove the word "immediately" from here since clicking the action button opens the modal rather than immediately starting generation.